client/src/modules/cohort/model/actions.js

```diff
@@ -214,7 +214,7 @@ export const updateCohort = (name, cohortId, data) => dispatch =>
     });
 
 export const deleteCohort = (cohortId, name) => dispatch =>
-  deleteData(`/api/cohorts/${cohortId}`)
+  deleteData(`/api/cohorts/${cohortId}/delete`)
```

Comment:
why delete in route? 
you are sending DELETE request so that server only handles DELETE request, not PATCH or any different